exit-what-2.md

```diff
@@ -3,6 +3,6 @@ layout: default
 title: Exit What 2
 ---
 
-# Exit what 2
+# Unfortunately, we can’t complete your request.
 
-Explanation of why not to proceed after the second what question.
+The Department of Health and Human Service’s works only with health information-related violations pertaining to HIPAA. 
```

Comment:
This is a dead end. Is there anything else we can provide them here in terms of next steps? Or at least clarifying (or restating) why we don't think it applies. Let's try to avoid just referring to "HIPPA" by itself and assume that people know that that means.